viadot/sources/cloud_for_customers.py

```diff
@@ -205,6 +241,8 @@ def to_df(self, fields: List[str] = None, if_empty: str = "warn") -> pd.DataFram
         """
         records = self.to_records()
         df = pd.DataFrame(data=records)
+        if df.empty:
+            logger.warning("The newly created dataframe is empty.")
```

Comment:
I would keep the convention we have of using camel case in this case (DataFrame).

Reply:
Corrected as suggested.